deps/rabbit/src/rabbit_stream_coordinator.hrl

```diff
@@ -59,10 +59,11 @@
 -record(rabbit_stream_coordinator, {streams = #{} :: #{stream_id() => #stream{}},
                                     monitors = #{} :: #{pid() => {stream_id() | %% v0 & v1
                                                                   #{stream_id() => ok}, %% v2
-                                                                  monitor_role()}},
+                                                                  monitor_role()} |
+                                                                 {pid(), sac}},
```

Comment:
In the case of SAC, map key and value have the same PID it seems. Why do we need to store the PID in the value?

Reply:
Yes, this can be simplified.